projpicker/web/index.html

```diff
@@ -0,0 +1,288 @@
+<!-- vim: set tabstop=4 shiftwidth=4 expandtab: -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>ProjPicker Desktop GUI Map</title>
+<meta charset="utf-8" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" crossorigin="" />
```

Comment:
Are all these `link` and `script` tags required? If so, let's make sure to download them all locally just in case the upstream devs break backward compatibility.